apps/api/src/common/services/email.service.ts

```diff
@@ -75,48 +124,72 @@ export class EmailService {
     projectId: string,
     userEmail: string,
     projectName: string,
-  ) {
+  ): Promise<{ success: boolean; error?: string }> {
+    const ch = this.requireChannel("default");
```

Comment:
not super critical, but I feel like these error objects should be more consistent - one has `ok: true` and one has `success: true` etc